templates/logging-sidecar-configmap.yaml

```diff
@@ -5,94 +5,33 @@
 kind: ConfigMap
 apiVersion: v1
 metadata:
-  name: {{ .Release.Name }}-logging-sidecar
+  name: sidecar-logging
   labels:
     tier: airflow
     component: logging-sidecar
     release: {{ .Release.Name }}
     chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
     heritage: {{ .Release.Service }}
 data:
-  main.conf: |-
-    <source>
-      @id unstructured-logging
-      @type tcp
-      tag tcp.events
-      bind 127.0.0.1
-      port 5170
-      <parse>
-        @type regexp
-        expression /(?<version>[^ ]+) (?<logtime>[^ ]+) (?<stream>[^ ]+) (?<component>[^ ]+) (?<message>.*)/
-      </parse>
-    </source>
+  vector-config.yaml: |
+    data_dir : #{ENV['SIDECAR_LOGS_DIR']}
+    sources:
+      generate_syslog:
+        type: file
+        include:
+          - #{ENV['SIDECAR_LOGS_DIR']}/*.log
+        read_from: beginning
+    sinks:
+      out:
+        type: elasticsearch
+        inputs:
+          - generate_syslog
+        mode: bulk
+        compression: none
+        endpoint: "http://{{ .Values.loggingSidecar.host }}:{{ .Values.loggingSidecar.port }}"
+        auth:
+          strategy: "basic"
+          user: {{ .Values.loggingSidecar.user }}
+          password : {{ .Values.loggingSidecar.password }}
```

Comment:
We might have to give first preference to global `customLogging` then elasticsearch service running in `astronomer` namespace.

FYI: We're using vector by datadog [here](https://vector.dev/docs/reference/configuration/sinks/elasticsearch/) is how the config will be mapped.

Reply:
@pgvishnuram this will be handled in houston itself right? Precedence given to global ES if custom logging enabled over default config?

Reply:
Yes correct. No extra configuration required